examples/unconditional_image_generation/train_unconditional_mlflow.py

```diff
@@ -677,89 +668,43 @@ def transform_images(examples):
 
             if (step) % args.logging_steps == 0:
                 logs = {"loss": loss.detach().item(), "lr": lr_scheduler.get_last_lr()[0], "step": global_step}
-                
                 interval_elapsed_time = time.time() - interval_start_time
                 interval_start_time = time.time()
-                interval_throughput = (args.logging_steps*args.train_batch_size) / interval_elapsed_time                  
+                interval_throughput = (args.logging_steps * args.train_batch_size) / interval_elapsed_time
                 throughput_list.append(interval_throughput)
-                interval_lr = lr_scheduler.get_last_lr()[0] 
+                interval_lr = lr_scheduler.get_last_lr()[0]
                 mlflow.log_metric('loss', loss.detach().item(), step=global_step)
                 mlflow.log_metric('throughput', interval_throughput, step=global_step)
                 mlflow.log_metric('lr', interval_lr, step=global_step)
 
                 if args.use_ema:
                     logs["ema_decay"] = ema_model.cur_decay_value
+
                 progress_bar.set_postfix(**logs)
                 accelerator.log(logs, step=global_step)
-            
+
             if len(throughput_list) == 5:
                 avg_throughput = sum(throughput_list[1:-1]) / (len(throughput_list) - 2)
                 epoch_time = len(dataset) / avg_throughput
                 mlflow.log_metric('avg_throughput', avg_throughput)
-                mlflow.log_metric('epoch_time', epoch_time)   
+                mlflow.log_metric('epoch_time', epoch_time)
 
             if global_step >= max_train_steps:
                 if len(throughput_list) < 5:
                     avg_throughput = sum(throughput_list) / len(throughput_list)
                 elif len(throughput_list) > 5:
                     avg_throughput = sum(throughput_list[1:-1]) / (len(throughput_list) - 2)
+
                 epoch_time = len(dataset) / avg_throughput
                 mlflow.log_metric('avg_throughput', avg_throughput)
                 mlflow.log_metric('epoch_time', epoch_time)
-                # elapsed_time = time.time() - start_time
-                # one_epoch_time = elapsed_time/ args.num_epochs
-                # throughput = (max_train_steps * args.train_batch_size * args.gradient_accumulation_steps) / elapsed_time
-
-                # mlflow.log_metric('avg_throughput', throughput)
-                # mlflow.log_metric('one_epoch_time', one_epoch_time)
                 break
 
         progress_bar.close()
-        
         accelerator.wait_for_everyone()
 
         # Generate sample images for visual inspection
         if accelerator.is_main_process:
-            # if epoch % args.save_images_epochs == 0 or epoch == args.num_epochs - 1:
```

Comment:
if `save_images_epochs` is still exposed as a cli arg then you should keep this code blocks. Otherwise it is ok to remove